### PR TITLE
Created Components and state function

### PR DESCRIPTION
**Pull request by Maxwell011**
### Hello in this PR I have done the following 👍 

- I have created the component folder.
- I have created React components.

Kindly Review 🙏

### CODE REVIEW

File: src/Components/calculator.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+// import { render } from '@testing-library/react';
```
Comment (Nemwel-Boniface) [inline src/Components/calculator.js:1]: - [ ] Kindly remove all commented-out, unused/ unwanted pieces of code from your codebase. It makes it look messy and cluttered.

**Review by Nemwel-Boniface — CHANGES_REQUESTED** [review]
## Hello @Maxwell011    :wave: , 
I'm Nemwel your reviewer for this PR :man_beard: 

Good job working on your project so far! :clap: 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️  


## ♻️ Required Changes ♻️

- [ ] As it is stated in the project requirements, you are supposed to delete all the boilerplate files created by CRA. I can currently still see the  App.test.js, reportwebvital.js, setuptests.js files which all contain boilerplate that you are not currently using. Kindly delete them,

_Check the comments under the review._

## [Optional] suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! :partying_face: 👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Maxwell011** [comment]
I have cleared unwanted files and cleared comments Kindly reviwed

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @Maxwell011👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**